src/libData/AccountData/LogEntry.cpp

```diff
@@ -22,7 +22,7 @@ bool LogEntry::Install(const Json::Value& eventObj,
                        const Address& address) //, unsigned int& numIndexed)
 {
     m_eventObj = eventObj;
-    if (!m_eventObj.isMember("eventname") || !m_eventObj.isMember("params"))
+    if (!m_eventObj.isMember("_eventname") || !m_eventObj.isMember("params"))
```

Comment:
I think for next time we can define these fields as constants somewhere to avoid these typos